EHospital.Authorization.BusinessLogic/EmailAction/EmailSender.cs

```diff
@@ -0,0 +1,12 @@
+﻿namespace EHospital.Authorization.BusinessLogic
+{
+    using System.Threading.Tasks;
+
+    public class EmailSender : IEmailSender
+    {
+        public Task SendEmailAsync(string email, string subject, string message)
+        {
+            return Task.CompletedTask;
```

Comment:
For such case you can just  throw new NotImplementedException(); and write TODO: .....